server/.gitignore

```diff
@@ -1,7 +1,8 @@
-/dsps.zip
-
 /coverage*
 /pprof
 
 # Temporary file in Makefile
 /go.mod.bak
+
+dist/
+CREDITS
```

Comment:
I recognize it's added because the output destination has changed, but is the `dsps` directory no longer needed?

Reply:
You're right, I deleted `dsps` directory, and changed Makefile slightly(this is still used, and convenient for building image locally).

Reply:
👍